agent/ffwd-local-pubsub-debug.yaml

```diff
@@ -0,0 +1,31 @@
+input:
+  plugins:
+    - type: json
+      protocol:
+        type: tcp
+    - type: json
+      protocol:
+        type: udp
+    - type: protobuf
+      protocol:
+        type: udp
+    - type: protobuf
+      protocol:
+        type: tcp
+    - type: riemann
+      protocol:
+        type: tcp
+    - type: riemann
+      protocol:
+        type: udp
+    - type: http
+      protocol:
+        type: tcp
+
+output:
+  plugins:
+    - type: debug
+    - type: pubsub
+      flushInterval: 10000
+      project: TEST
```

Comment:
one nitpick - would be good to keep the project and topic consistent with whats in the readme...
https://github.com/spotify/ffwd/tree/master/modules/pubsub